core/consensus/grandpa/structs.hpp

```diff
@@ -9,20 +9,20 @@
 #include <boost/asio/steady_timer.hpp>
 #include "common/blob.hpp"
 #include "common/wrapper.hpp"
+#include "crypto/ed25519_types.hpp"
+#include "primitives/common.hpp"
 
 namespace kagome::consensus::grandpa {
 
   using Timer = boost::asio::basic_waitable_timer<std::chrono::steady_clock>;
 
-  using BlockHash = common::Wrapper<common::Hash256, struct BlockHashTag>;
-  using BlockNumber = common::Wrapper<uint64_t, struct BlockNumberTag>;
   using RoundNumber = common::Wrapper<uint64_t, struct RoundNumberTag>;
 
   /// voter identifier
-  using Id = common::Wrapper<common::Hash256, struct IdTag>;
+  using Id = primitives::AuthorityId;
 
   /// voter signature
-  using Signature = common::Wrapper<std::vector<uint8_t>, struct SignatureTag>;
+  using Signature = crypto::ED25519Signature;
```

Comment:
i would leave signature as is, just plain vector. because in this case the type is not specified (sr/ed...), but it still can be processed.

Reply:
If we move from ED to some other crypto we will just update this using